packages/e2e-tests/specs/blocks/table.test.js

```diff
@@ -3,6 +3,8 @@
  */
 import { createNewPost, insertBlock, getEditedPostContent } from '@wordpress/e2e-test-utils';
 
+const buttonSelector = "//div[@data-type='core/table']//button[text()='Create Table']";
```

Comment:
nit: Renaming this to `createButtonSelector` might avoid confusion down the track if there's ever another button.